lib/text_diff.ml

```diff
@@ -1,174 +1,245 @@
 type conflict2 = { orig : string list; new_ : string list }
-type hunk = Change of conflict2 | Same of string
+type hunk = Change of conflict2 * bool | Same of string * bool
```

Comment:
Arguments whose role is not explicit from the type should always be named. You can use a record here to name the fields!